pkg/scheduler/plugins/deviceshare/pod_handler.go

```diff
@@ -46,6 +46,11 @@ func (n *nodeDeviceCache) onPodAdd(obj interface{}) {
 		return
 	}
 
+	if pod.Spec.NodeName == "" {
+		klog.Errorf("failed to get node name from pod %v", klog.KObj(pod))
```

Comment:
There is no need to add logs here, pod.spec.nodeName is normal.  method `onPodDelete` also need to check the spec.nodeName

Reply:
fixed